lib/App.js

```diff
@@ -2178,6 +2244,18 @@ class App {
       await NpmCommands.install({ saveDev: true, path: appPath }, {
         id: '@types/homey@npm:homey-apps-sdk-v3-types'
       });
+
+      if (App.usesTypeScript({ appPath })) {
+        await NpmCommands.install({ save: true, path: appPath }, {
```

Comment:
I think we also want to install `typescript` as a dev dependency (i believe that is the recommended way to use typescript)